webapps/frontend/public/app/admin/locales/de.json

```diff
@@ -66,8 +66,10 @@
     "AUTHORIZATION_TYPE": "Typ",
     "AUTHORIZATION_UPDATE": "Autorisierung konnte nicht aktualisiert werden",
     "AUTHORIZATION_USER": "Benutzer",
+    "AUTHORIZATION_USER_AUTH_INFO": "Autorisierungen werden nicht automatisch gelöscht, wenn Sie den Benutzer löschen. Übrig gebliebene Autorisierungen werden wieder angewendet, wenn ein Benutzer mit der gleichen ID angelegt wird. Nutzen Sie stattdessen 'Gruppen' Autorisierungen.",
     "AUTHORIZATION_USER_GROUP": "Benutzer / Gruppe",
-    "AUTHORIZATION_USER_GROUP_ID": "Benutzer / Gruppen-ID",
+    "AUTHORIZATION_USER_ID": "User ID",
+    "AUTHORIZATION_GROUP_ID": "Group ID",
```

Comment:
❓These don't need to be translated?
I'm asking because the previous text was translated: `"AUTHORIZATION_USER_GROUP_ID": "Benutzer / Gruppen-ID"`.

Reply:
Nice catch! Just overlooked it.